core/capabilities/ccip/ocrimpls/no_op_transmitter.go

```diff
@@ -0,0 +1,47 @@
+package ocrimpls
+
+import (
+	"context"
+
+	"github.com/smartcontractkit/libocr/offchainreporting2plus/ocr3types"
+	"github.com/smartcontractkit/libocr/offchainreporting2plus/types"
+
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+)
+
+var _ ocr3types.ContractTransmitter[[]byte] = (*noOpTransmitter)(nil)
+
+// NewNoOpTransmitter creates a new no-op transmitter. It is intended to be used in
+// role DONs where the node that is participating cannot transmit to the destination chain.
+func NewNoOpTransmitter(lggr logger.Logger) *noOpTransmitter {
+	return &noOpTransmitter{
+		lggr: lggr.Named("NoOpTransmitter"),
+	}
+}
+
+// noOpTransmitter is an implementation of ocr3types.ContractTransmitter[[]byte]
+// that does nothing. It is intended to be used in role DONs where the
+// node that is participating cannot transmit to the destination chain.
+type noOpTransmitter struct {
+	lggr logger.Logger
+}
+
+// FromAccount implements ocr3types.ContractTransmitter.
+func (n *noOpTransmitter) FromAccount(context.Context) (types.Account, error) {
+	n.lggr.Errorw("no-op transmitter FromAccount() called, it shouldn't be!")
```

Comment:
Maybe crit level if it must not be called?

Reply:
Maybe and also I'll add more logging context to make the debug easier

Reply:
Bumped to crit and added more ctx.